README.md

```diff
@@ -183,14 +183,14 @@ We support exclusion of vulnerability either by CVE-ID (ex: `CVE-2018-20303`) or
 
 ##### Via CLI flag
 * `./nancy -exclude-vulnerability CVE-789,bcb0c38d-0d35-44ee-b7a7-8f77183d1ae2 /path/to/your/Gopkg.lock`
-* `./nancy -exclude-vulnerability CVE-789,bcb0c38d-0d35-44ee-b7a7-8f77183d1ae2 /path/to/your/go.sum`
+* `go list -json -m all | ./nancy -exclude-vulnerability CVE-789,bcb0c38d-0d35-44ee-b7a7-8f77183d1ae2`
```

Comment:
Likely `--exclude-vulnerability` I think

Reply:
> Likely `--exclude-vulnerability` I think

Yes! Also fixed some other inconsistencies in: 25c11c1